scraper/src/main/java/com/example/cosc345/scraper/scrapers/woocommerce/TastiBitzScraper.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.cosc345.scraper.scrapers.woocommerce
+
+import com.example.cosc345.scraper.scrapers.generic.WooCommerceScraper
+import com.example.cosc345.shared.models.Region
+import com.example.cosc345.shared.models.Retailer
+import com.example.cosc345.shared.models.Store
+
+/**
+ * The Tasti Bitz specific implementation of the [WooCommerceScraper], which essentially just passes information through to it.
+ */
```

Comment:
@constructor Create a new instance of this scraper.

Reply:
See above